doc/api/net.md

```diff
@@ -331,10 +331,18 @@ Returns `server`.
 added: v0.3.4
 -->
 
-This object is an abstraction of a TCP or local socket.  `net.Socket`
-instances implement a duplex Stream interface.  They can be created by the
-user and used as a client (with [`connect()`][]) or they can be created by Node.js
-and passed to the user through the `'connection'` event of a server.
+This class is an abstraction of a TCP socket, a UNIX domain socket, or
+a Windows named pipe. A `net.Socket` is also a [duplex stream][], so it
+can be both readable and writable, and it is also a [`EventEmitter`][].
+
+A `net.Socket` can be created by the user and used as an abstraction
```

Comment:
"used as an abstraction of the client" is a bit convoluted in my opinion, consider: "used directly to connect to a server". Almost all programming objects are in some sense "abstractions" (a url.URL is an abstraction of a "URL", a js-String is an abstraction of a "sequence of characters", etc., but its not how we usually describe them).